release/models/aft/openconfig-aft-common.yang

```diff
@@ -385,6 +407,37 @@ submodule openconfig-aft-common {
     }
   }
 
+  grouping aft-common-entry-nexthop-gre-state {
+    description
+      "GRE encapsulation applied on a next-hop";
+
+    leaf src-ip {
```

Comment:
To be clear here, GRE over both v6 and v4 are intended to be covered by this same modelling? That sounds good to me, but I just wanted to verify.

Reply:
Yes, this is correct. Do you want me to update the `description` with that as well?

Reply:
Please do update the description since the question was asked.  It's worth it since the question came up here.  :)